package.json

```diff
@@ -10,7 +10,8 @@
   "lint": "jshint lib test",
   "start": "node index.js",
   "test": "mocha",
-  "audit": "./node_modules/.bin/audit-ci --moderate --registry=https://registry.npmjs.org"
+  "audit": "./node_modules/.bin/audit-ci --moderate --registry=https://registry.npmjs.org",
+  "build-doc": "apidoc-openapi --project ./apidoc.json --src lib --out openapi.json"
```

Comment:
I would add "@forfuture/apidoc-openapi": "^0.0.1" as a dev dependancy and save the json file under doc directly so there is no need to change artifact_node.sh